test/parallel/test-http-response-splitting.js

```diff
@@ -46,8 +48,7 @@ const server = http.createServer((req, res) => {
     default:
       assert.fail('should not get to here.');
   }
-  if (count === 3)
-    server.close();
+  countdown.dec();
```

Comment:
You can remove the `count` variable itself, if we can return the current value from `Countdown.dec`

Reply:
@thefourtheye : For what you've suggested.. the `dec()`  returns the remaining count and not current count right... `return this[kLimit]`
So I guess we would need the count variable being used in switch else I'll have to do `limit - countdown.remaining()` or something to get the current count..Correct me if I'm wrong... we're counting down right...!

Reply:
That's okay. That can be done in a separate PR though. I think that would be helpful.

Reply:
@thefourtheye : I'll have it done over another PR !